07-routing-gossip.md

```diff
@@ -323,6 +323,9 @@ any future fields appended to the end):
     - MUST NOT process the message further.
   - if `features` field contains _unknown even bits_:
     - SHOULD NOT connect to the node.
+    - Unless paying a [BOLT #11](11-payment-encoding.md) invoice which does not
```

Comment:
I don't get why you're adding those two points.
Since the first bullet point states that you shouldn't be connected in the first place, you won't ever route to that node.
If you connect anyway (because line 325 says `SHOULD NOT` and not `MUST NOT`), you're accepting the risk of some undefined behavior, and I don't see why you wouldn't route messages through/to that node.
Can you clarify with some examples?